py-geopolars/pyproject.toml

```diff
@@ -7,8 +7,7 @@ name = "geopolars"
 version = "0.1.0"
 dependencies = [
   "polars",
-  "numpy >= 1.16.0",
-  "typing_extensions >= 4.0.0; python_version < '3.10'",
+  "oldest-supported-numpy",
```

Comment:
Actually here these are runtime dependencies, not build dependencies, so we can go back to standard `numpy`. It's also not clear if we need numpy as a dependency, but we can leave it for now